projects/storefrontapp-e2e-cypress/cypress/integration/login-e2e-spec.ts

```diff
@@ -0,0 +1,59 @@
+import { user } from '../sample-data/big-happy-path';
+import { register, login } from '../helpers/auth-forms';
+
+describe('Login', () => {
+  before(() => {
+    // Go to Home
+    cy.window().then(win => win.sessionStorage.clear());
+    cy.visit('/');
+
+    // Register a new user.
+    cy.get('cx-login [role="link"]').click();
+    cy.get('cx-page-layout')
+      .getByText('Register')
+      .click();
+    register(user);
+
+    // check if the the user exist in the dom
+    cy.get('cx-login .cx-login-status__greet').should('exist');
```

Comment:
```suggestion
```

This assertion is not needed. Next assertion would fail if the element doesn't exist.